src/index.html

```diff
@@ -1,31 +1,29 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>2048</title>
+    <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The stylesheet link references a `.scss` file. Ensure that this file is being preprocessed into a `.css` file, or link directly to the compiled CSS file instead.